Makefile

```diff
@@ -45,7 +45,7 @@ kube-delete-database:
 
 kube-delete-all: kube-delete kube-delete-database
 
-kube-create:
+kube-create: update-manifests
```

Comment:
I believe this was intentional to not automatically run update-manifests each time one ran kube-create, but I could be wrong.